frontend/www/js/omegaup/course/edit.js

```diff
@@ -450,18 +450,22 @@ OmegaUp.on('ready', function() {
             if (ev.participants !== '')
               participants = ev.participants.split(',');
             if (ev.participant !== '') participants.push(ev.participant);
-            let promises = participants.map(function(participant) {
-              return API.Course.addStudent({
-                course_alias: courseAlias,
-                usernameOrEmail: participant.trim()
+            if (participants.length > 0) {
```

Comment:
Aquí puedes usar el [guard clause pattern](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html).

```suggestion
            if (participants.length == 0) {
              UI.error(T.wordsEmptyAddStudentInput);
              return;
            }
            // El resto de la función.
```

Reply:
Creo que ya:)